cmd/minikube/cmd/cache.go

```diff
@@ -38,8 +38,8 @@ const allFlag = "all"
 // cacheCmd represents the cache command
 var cacheCmd = &cobra.Command{
 	Use:   "cache",
-	Short: "Add, delete, or push a local image into minikube",
-	Long:  "Add, delete, or push a local image into minikube",
+	Short: "Add an image into minikube as a local cache, or delete, reload the cached images",
```

Comment:
I suggest better wording and while we are at it, lets have different short and long descriptions

Reply:
I agree. How about the following as Short?
"Cache images into minikube, or delete, reload them"
or more simply
"Manage cache for images"

FYI, Long is output as the command usage. Both Short and Long are used in the document site.